.travis.yml

```diff
@@ -6,20 +6,24 @@ go:
 services:
   - rabbitmq
 
+sudo: required
```

Comment:
Apparently it's default, unless you set `sudo: required`, or call sudo. At least that's how I [read this.](http://docs.travis-ci.com/user/workers/container-based-infrastructure/)